experimental/dds/tree2/src/test/shared-tree/schematizeTree.spec.ts

```diff
@@ -3,41 +3,110 @@
  * Licensed under the MIT License.
  */
 import { strict as assert } from "assert";
-import { MockFluidDataStoreRuntime } from "@fluidframework/test-runtime-utils";
-import { SchemaBuilder, Any } from "../../feature-libraries";
-import { SharedTreeFactory } from "../../shared-tree";
-import { ValueSchema, AllowedUpdateType, storedEmptyFieldSchema } from "../../core";
+import {
+	MockFluidDataStoreRuntime,
+	validateAssertionError,
+} from "@fluidframework/test-runtime-utils";
+import { SchemaBuilder, Any, TypedSchemaCollection, FieldSchema } from "../../feature-libraries";
+import { ISharedTreeView, SharedTreeFactory } from "../../shared-tree";
+import {
+	ValueSchema,
+	AllowedUpdateType,
+	storedEmptyFieldSchema,
+	SimpleObservingDependent,
+} from "../../core";
 import { typeboxValidator } from "../../external-utilities";
+import { TestTreeProviderLite } from "../utils";
 
 const factory = new SharedTreeFactory({ jsonValidator: typeboxValidator });
-const builder = new SchemaBuilder("Schematize Tree Tests");
 
+const builder = new SchemaBuilder("Schematize Tree Tests");
 const root = builder.leaf("root", ValueSchema.Number);
 const schema = builder.intoDocumentSchema(SchemaBuilder.fieldOptional(Any));
 
 const builderGeneralized = new SchemaBuilder("Schematize Tree Tests Generalized");
 const rootGeneralized = builderGeneralized.leaf("root", ValueSchema.Serializable);
```

Comment:
Oh, I see, the builder doesn't need you to pass it back in for it to be made part of the schema. It normally would be passed back in but you don't care here for those tests. That's a bit confusing but I'm not sure it matters to much.

Reply:
I'm not sure what you mean by "pass it in". Pass the schema into what? Its perfectly reasonable to have a schema thats not mentioned in any other schema. Thats simply the nature of open polymorphism.

The fact that these tests happen to both not refer to these schema from other schema, and not refer to them directly in the code since the nodes are typed implicitly using contextually typed node data is a bit of a coincidence, but nothing actually strange about it.